pkg/codegen/utils.go

```diff
@@ -999,3 +999,15 @@ func isAdditionalPropertiesExplicitFalse(s *openapi3.Schema) bool {
 
 	return *s.AdditionalProperties.Has == false //nolint:gosimple
 }
+
+func generateReposneName(responseName, operationID string) string {
+	if !unicode.IsDigit(rune(responseName[0])) {
```

Comment:
```suggestion
	if len(responseName>0) && !unicode.IsDigit(rune(responseName[0])) {
```

to avoid potential panic if someby wrotes response with empty string as a status